### ISSUE TITLE
Multiple irrelevant commits

### ISSUE THREAD

**Issue by Tekipeps** [Tekipeps/aul-forum#17]
@BidemiEnoch you could squash commits into one if they dont describe a single, logical, coherent change

https://www.internalpointers.com/post/squash-commits-into-one-git
https://stackoverflow.com/a/5721879

**Comment by BidemiEnoch** [comment]
Nah, i forgot to checkout to a dummy branch, i wasnt really changing anything, just testing the pre commit hook so i was using irrelevant commit messages, fixed that, I guess

**Comment by Tekipeps** [comment]
alright
